utils/zerovec/src/ule/chars.rs

```diff
@@ -0,0 +1,108 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! ULE implementation for the `char` type.
+
+use super::*;
+use std::convert::TryFrom;
+
+/// A u8 array of little-endian data corresponding to a Unicode code point.
+///
+/// The bytes of a `CharULE` are guaranteed to represent a little-endian-encoded u32 that is a
+/// valid `char` and can be converted without validation.
```

Comment:
`request`: Can we get example here? This crate is non-trivial and I'd prefer over-example docs than under to save us from having to load the whole thing into mind every time we touch it.